### PR TITLE
Added documentation for CaseSensitivity in commandApp.md and settings.md

### LINKED ISSUE #653: CaseSensitivity should be documented somewhere
As it affects `Commands` and long `Options` it should probably be in https://spectreconsole.net/cli/commandapp and https://spectreconsole.net/cli/settings

### PR DESCRIPTION
**Pull request by alex-eshock**
Fixes #653 where CaseSensitivity has now been documented in the respective .md files.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/spectreconsole/spectre.console?pullRequest=1042) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/spectreconsole/spectre.console?pullRequest=1042) before we can accept your contribution.<br/><br/>:x: alex-eshock [sign now](https://cla.dotnetfoundation.org/spectreconsole/spectre.console?pullRequest=1042)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/spectreconsole/spectre.console?pullRequest=1042) it.</sub>

**Comment by alex-eshock** [comment]
Hold on I made a mistake. I'm going to close the pull request and create a new one.

**Comment by alex-eshock** [comment]
Fixes https://github.com/spectreconsole/spectre.console/issues/653 where CaseSensitivity has now been documented in the respective .md files.

File: docs/input/cli/commandApp.md [hunk 1]
```diff
@@ -21,6 +21,10 @@ app.Configure(config =>
 });
 ```
 
+## Case Sensitivity
+
+Commands are in `PascalCase`, also known as `UpperCamelCase`.
+
```
Comment (patriksvensson) [inline docs/input/cli/commandApp.md:27]: I think this doesn't seem right. Commands that we provide to an application, such as `myexe.com mycommand` are case sensitive in the sense that `myexe.com Mycommand` (notice the capital 'M') would not match the configured command.

There is no preference when it comes to naming of commands.

File: docs/input/cli/settings.md [hunk 2]
```diff
@@ -24,6 +24,10 @@ public sealed class MyCommandSettings : CommandSettings
 
 This setting file tells `Spectre.Console.Cli` that our command has two parameters. One is marked as a `CommandArgument`, the other is a `CommandOption`.
 
+## Case Sensitivity
+
+CommandsSettings are in `PascalCase`, also known as `UpperCamelCase`.
+
```
Comment (patriksvensson) [inline docs/input/cli/settings.md:30]: I think this doesn't seem right. Options that we provide to an application such as `myexe.com --myoption 1` are case sensitive in the sense that `myexe.com --Myoption 1` (notice the capital 'M') would not match the configured option.

There is no preference when it comes to naming of settings.

**Review by patriksvensson — CHANGES_REQUESTED** [review]

**Pull request closed without merging**